src/subtitles/imscsubtitles.js

```diff
@@ -83,7 +83,10 @@ function IMSCSubtitles(mediaPlayer, autoStart, parentElement, mediaSources, defa
         }
 
         try {
-          const xml = fromXML(responseText.split(/<\?xml[^?]+\?>/i)[1] || responseText)
+          const xmlText = isSubtitlesWhole()
+            ? responseText.replace(/^.*<\?xml[^?]+\?>/i, "")
+            : responseText.split(/<\?xml[^?]+\?>/i)[1] || responseText
```

Comment:
What's the reason we split for live? Do you reckon it'll reduce test effort?

Reply:
No idea, intention for this is to reduce the blast radius of the change and limit it to VOD. There isn't really a performance issue with running split like this on segmented streams because they're only short.

Reply:
Yea fair – one for us to look into the performance of this further